src/core/values/routes.ts

```diff
@@ -4,4 +4,14 @@ export enum Routes {
   SignUp = '/sign-up',
   ResetPassword = '/reset-password',
   Confirmation = '/confirm',
-}
\ No newline at end of file
+  Catalog = '/catalog',
+  Profile = '/profile',
+}
```

Comment:
Enum is a set of possible options variable can be equal to, so for better semantics it's worth using singular form:
```typescript
enum Route {
  Home = '/',
  SignIn = '/sign-in',
  SignUp = '/sign-up',
  ResetPassword = '/reset-password',
  Confirmation = '/confirm',
} 
```

In this concrete case of `Route` enum means that you might have some `route` variable that must be equal to one of the members of the `Route` enum